background/load-addon-manifests.js

```diff
@@ -66,7 +66,7 @@ const localizeSettings = (addonId, setting, tableId) => {
       } else {
         const file = await (await fetch(`/addons/${addonId}/addon.json`)).json();
         manifest = file;
-        newCache[addonId] = file;
+        newCache[addonId] = JSON.parse(JSON.stringify(file));
```

Comment:
Note that `structuredClone()` worked equivalently but isn't supported by Chrome 96.

Reply:
😮 That's a feature I didn't know!

`storage.session` is only supported in Chrome 102+ and Firefox 115+, so actually, we wouldn't even have to use `structuredClone()` in older versions, so technically we could replace `JSON.parse(JSON.stringify(obj))` with `structuredClone()`. We could also fall back to `JSON.parse(JSON.stringify(obj))` if `structuredClone()` is not available, or we could use [a polyfill](https://github.com/ungap/structured-clone).

It seems to be a little faster to use `structuredClone()`.
![console](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/1dc940e4-c038-40c8-8661-439fa673cc3c)

Reply:
Average of about 0.58772786458 ms difference and a percent increase of about 62.658337671%. Expression used: |((0.994873046875+0.989990234375+0.8291015625))-((1.7451171875+1.5361328125+1.2958984375))|/((0.994873046875+0.989990234375+0.8291015625))

Reply:
I tried with an even larger object and the time save was really substantial - `JSON.parse(JSON.stringify())` took about a quarter of a second while `structuredClone()` only took, like, 4 ms.

It's a little more complicated than that, though - cloning a smaller amount of data, like an object that takes only a millisecond or two to clone, `JSON.parse(JSON.stringify())` pulls ahead, surprisingly, which can add up when you're cloning hundreds of small objects.

It's possible this behavior will also vary from browser to browser and platform to platform.

Reply:
250 ms to 4 ms??! That must mean the JSON method grows almost exponentially in time as the object gets larger.

Reply:
> `storage.session` is only supported in Chrome 102+ and Firefox 115+, so actually, we wouldn't even have to use `structuredClone()` in older versions

Huh, good point, I missed that.
You may submit a PR to change it, if you want. But make sure to add a code comment or a useless if statement, so that it's clear that structuredClone should not be used in other contexts.